package.json

```diff
@@ -35,6 +36,7 @@
     "@jscad/sample-files": "github:jscad/sample-files#master",
     "ava": "1.4.0",
     "lerna": "3.19.0",
+    "nyc": "^15.0.0",
```

Comment:
we should always use exact versions (so without the caret "^" symbol); as not exact versions of dependencies can lead to some issues (breakeage, unanonced and not correct api changes etc)

Reply:
done